engine/src/main/scala/cromwell/engine/io/IoActor.scala

```diff
@@ -170,12 +170,19 @@ object IoActor {
     503
   )
   
+  val AdditionalRetryableErrorMessages = List(
+    "Connection closed prematurely"
+  )
+  
   /**
     * Failures that are considered retryable.
     * Retrying them should increase the "retry counter"
     */
   def isRetryable(failure: Throwable): Boolean = failure match {
-    case gcs: StorageException => gcs.isRetryable || AdditionalRetryableHttpCodes.contains(gcs.getCode) || isRetryable(gcs.getCause)
+    case gcs: StorageException => gcs.isRetryable || 
+      AdditionalRetryableHttpCodes.contains(gcs.getCode) || 
+      isRetryable(gcs.getCause) ||
+      AdditionalRetryableErrorMessages.exists(_.contains(gcs.getMessage))
```

Comment:
Shouldn't this be `[...].exists(gcs.getMessage.contains(_))`? It seems like the list we store is substrings of messages that they might return?

Reply:
Also do we want to throw in an `ignoreCase`?